### PR TITLE
Expand quick start

### PR DESCRIPTION
**Pull request by vishakha041**
This should be totally self contained in starting and using ApertureDB.

### CODE REVIEW

**Comment by vishakha041** [comment]
@gsaluja9  - we are missing the loader scripts. Let's merge those examples first, then we can finish this.

@bovlb  - you can review the database and set up sections on this branch. it will depend on quick start but we can fill in the rest.

What do you guys think of this organization now ... @drewaogle

File: notebooks/QuickStart.ipynb [hunk 1]
```diff
@@ -60,49 +89,110 @@
...
    "metadata": {},
    "source": [
-    "## Create and activate server configuration\n",
+    "### d) Add server configuration to secrets\n",
+    "\n",
+    "The easiest way to configure one or more Colab notebooks to point to the same ApertureDB server is to put the JSON config into a Colab secret.  \n",
+    "This secret will persist over kernel restarts and can be used by all of your notebooks, \n",
+    "but it will not be exposed if you share your notebook with anyone.\n",
+    "\n",
+    "![Screenshot of setting a Colab Secret](images/colab-secret.png)\n",
```
Comment (gsaluja9) [inline notebooks/QuickStart.ipynb:98]: I feel adding these steps to all the notebooks will make them too big. This should be added to setup notebooks colab section. https://docs.aperturedata.dev/Setup/client/notebooks .

Comment (vishakha041) [reply notebooks/QuickStart.ipynb:98]: am not putting them in all - putting a reference to this notebook though. Quick start - self contained - get people going. others - we assume they are set up already.

Comment (bovlb) [reply notebooks/QuickStart.ipynb:98]: If I put this information in [/Setup/client/notebooks](https://docs.aperturedata.dev/Setup/client/notebooks) , should it augment or replace the instructions already there?

Comment (gsaluja9) [reply notebooks/QuickStart.ipynb:98]: Those should be replaced.

**Pull request merged**